### ISSUE TITLE
Add support for Ghostty terminal

### ISSUE THREAD

**Issue by chianyungcode** [aws/amazon-q-developer-cli#323]
Add support for Amazon Q in [Ghostty](https://ghostty.org/)

**Comment by grant0417** [comment]
Im working on adding support in #335, also where did all of you see this issue? Seems there are a lot of 👍

**Comment by grant0417** [comment]
<img width="824" alt="Screenshot 2025-01-01 at 3 22 06 AM" src="https://github.com/user-attachments/assets/23ffb226-58ec-4a3c-89dc-ec9dd52ad8b7" />

It is working, this will be supported in the next release 🚀

**Comment by matheusrocha89** [comment]
@grant0417 That's great news. Looking forward for that.

**Comment by chianyungcode** [comment]
Is there any estimated time of arrival for the 1.6.0 release? @grant0417

**Comment by grant0417** [comment]
Sometime next week

**Comment by grant0417** [comment]
The 1.6.0 of Amazon Q has been released with support for Ghostty 🎉

**Comment by chianyungcode** [comment]
![CleanShot 2025-01-10 at 02 33 07@2x](https://github.com/user-attachments/assets/e31efd78-24b0-4855-89a8-f13b9bf30034)

**Comment by grant0417** [comment]
@chianyungcode Interesting, could you open another ticket for this, I am not having this issue as shown in the screen shot I posted above.

**Comment by sewalsh** [comment]
> Hi, the autocomplete looks like it's asymmetrical and covers the text behind it @grant0417

Try running `q integrations install input-method` and restart Ghostty.

**Comment by chianyungcode** [comment]
> @chianyungcode Interesting, could you open another ticket for this by running `q issue`, I am not having this issue as shown in the screen shot I posted above.

#395 has already opened a new ticket for that issue.

**Comment by chianyungcode** [comment]
> > Hi, the autocomplete looks like it's asymmetrical and covers the text behind it @grant0417
> 
> Try running `q integrations install input-method` and restart Ghostty.

<img width="260" alt="image" src="https://github.com/user-attachments/assets/ad73ac0d-286c-437a-bd92-b958964461f4" />

It is safe to allowing this?

**Comment by sewalsh** [comment]
Errr, I can guarantee that `CodeWhisperInputMethod` is already phoning all your data back to Amazon. 

What sort of 'privacy first' company do you think Amazon is?!

**Comment by grant0417** [comment]
Yes that is safe, we should rename the bundle but that is correct.

The only thing we use the input method for is getting the cursor position. You can audit all of the code for that here, there is no telemetry as part of the input method: https://github.com/aws/amazon-q-developer-cli/tree/main/crates/fig_input_method

**Comment by TTSKarlsson** [comment]
```bash
$ q integrations install input-method
error: Input method integration is only supported on macOS
```

This seems to be vendor locked to specific Apple products
